lightrag/kg/qdrant_impl.py

```diff
@@ -491,39 +525,23 @@ async def drop(self) -> dict[str, str]:
         """
         async with get_storage_lock():
             try:
-                # Delete the collection and recreate it
-                exists = False
-                if hasattr(self._client, "collection_exists"):
-                    try:
-                        exists = self._client.collection_exists(self.final_namespace)
-                    except Exception:
-                        exists = False
-                else:
-                    try:
-                        self._client.get_collection(self.final_namespace)
-                        exists = True
-                    except Exception:
-                        exists = False
-
-                if exists:
-                    self._client.delete_collection(self.final_namespace)
-
-                # Recreate the collection
-                QdrantVectorDBStorage.create_collection_if_not_exist(
```

Comment:
No need to re-create it.
Since it will done when `initialize()` is called again.